dotCMS/src/main/webapp/html/portlet/ext/contentlet/edit_contentlet_js_inc.jsp

```diff
@@ -683,7 +681,8 @@
             }
 
             var license = <%=LicenseUtil.getLevel()%>;
-            if ( license <= LicenseLevel.STANDARD.level ||  fieldRelatedData['fileName'].toLowerCase().endsWith("svg")){
```

Comment:
Question, the license levels should be handle in the client side, rather than the BE?

Reply:
@jdotcms BE, which needs to be taken into consideration when rewriting this piece in angular